ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -47,104 +134,61 @@ def call(BUILD_BRANCH, CREDENTIALS_ID, CODECOV_CREDENTIALS_ID) {
 						withCredentials([string(credentialsId: "${CODECOV_CREDENTIALS_ID}", variable: 'codecovToken')]) {
 							CODECOV_TOKEN = codecovToken
 						}
+					}
 
+					stage("Check") {
 						ansiColor('xterm') {
-						sh """
-						export GOPATH=${WORKSPACE}/go
-						export PATH=${WORKSPACE}/go/bin:\$PATH
-						if ! hash hg 2>/dev/null; then
-							sudo yum install -y mercurial
-						fi
-						hg --version
-						make check-setup
-						make check
-						if [ ${BUILD_BRANCH} == "master" ]
-						then
-							make test GO_COVER=y
-							curl -s https://codecov.io/bash | bash -s - -t ${CODECOV_TOKEN} || echo'Codecov did not collect coverage reports'
-						else
-							make test
-						fi
-						make
-						make e2e-build
-						"""
+							sh """
+							export GOPATH=${WORKSPACE}/go
+							export PATH=${WORKSPACE}/go/bin:\$PATH
+							# hq is required to clone git repos from https://bitbucket.org 
+							if ! hash hg 2>/dev/null; then
```

Comment:
Is `hg` needed? I don't have `hg` installed on my local machine and haven't got any issues yet.

Reply:
removed, `hg` is not needed anymore